#! /bin/sh /usr/share/dpatch/dpatch-run
## 002-nant-load-partial-name.dpatch by Jelmer Vernooij <jelmer@samba.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to avoid now obsolete function LoadWithPartialName() as 
## DP: the nant build file treats all warnings as errors.

@DPATCH@
diff -ur nant-0.85/src/NAnt.DotNet/Tasks/ScriptTask.cs nant-0.85-fixed/src/NAnt.DotNet/Tasks/ScriptTask.cs
--- nant-0.85/src/NAnt.DotNet/Tasks/ScriptTask.cs	2006-09-13 18:31:26.000000000 +0200
+++ nant-0.85-fixed/src/NAnt.DotNet/Tasks/ScriptTask.cs	2008-05-26 21:12:13.000000000 +0200
@@ -516,7 +516,7 @@
         #region Private Static Methods
 
         private static CodeDomProvider CreateCodeDomProvider(string typeName, string assemblyName) {
-            Assembly providerAssembly = Assembly.LoadWithPartialName(assemblyName);
+            Assembly providerAssembly = Assembly.Load(assemblyName);
             if (providerAssembly == null) {
                 throw new ArgumentException(string.Format(CultureInfo.InvariantCulture,
                     ResourceUtils.GetString("NA2037"), assemblyName));
Only in nant-0.85-fixed/src/NAnt.DotNet/Tasks: ScriptTask.cs~
